coral/src/app/features/topics/details/history/TopicHistory.tsx

```diff
@@ -0,0 +1,83 @@
+import { DataTable, DataTableColumn, EmptyState } from "@aivenio/aquarium";
+import { useTopicDetails } from "src/app/features/topics/details/TopicDetails";
+import { TopicOverview } from "src/domain/topic";
+
+interface TopicHistoryRow {
+  id: number;
+  logs: string; //Remarks
+  team: string; //
+  requestedBy: string;
+  requestedTime: string;
+  approvedBy: string;
+  approvedTime: string;
+}
+
+function TopicHistory() {
+  const { topicOverview } = useTopicDetails();
+
+  const columns: Array<DataTableColumn<TopicHistoryRow>> = [
+    {
+      type: "status",
+      field: "logs",
+      headerName: "Logs",
+      status: ({ logs }) => ({
+        status: "neutral",
+        text: logs,
+      }),
+    },
+    { type: "text", field: "team", headerName: "Team" },
+    { type: "text", field: "requestedBy", headerName: "Requested by" },
+    {
+      type: "text",
+      field: "requestedTime",
+      headerName: "Requested time",
```

Comment:
I think we agreed with Harshini about the "Requested on" / "Approved on" kind of copy. At least it's what is visible in our other tables with timestamps ^^

Reply:
💡  will update!